.zuul/playbooks/containerd-build/run.yaml

```diff
@@ -0,0 +1,25 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: config-golang
+      arch: arm64
+  tasks:
+    - name: Build containerd
+      shell:
+        cmd: |
+          set -xe
+          apt-get update
+          apt-get install -y btrfs-tools libseccomp-dev git pkg-config
+          echo $PATH
+          echo $GOPATH
+          mkdir -p $GOPATH/src/github.com/containerd/
+          ln -s '{{ ansible_user_dir }}/{{ zuul.project.src_dir }}' $GOPATH/src/github.com/containerd/containerd
+          cd $GOPATH/src/github.com/containerd/containerd
```

Comment:
Please add a NOTE for the three lines, only forked golang project have to do this workaround, it's not OpenLab limits.
```
          mkdir -p $GOPATH/src/github.com/containerd/
          ln -s '{{ ansible_user_dir }}/{{ zuul.project.src_dir }}' $GOPATH/src/github.com/containerd/containerd
          cd $GOPATH/src/github.com/containerd/containerd
```

Reply:
done

Reply:
Thank you.